projects/Four In A Row - Classic/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Four In A Row - Classic",
+    "description": "You can now play the classic Four In A Row board game on your phone or tablet! The objective of the game is to connect 4 of your chips in a row. You can do this horizontally, vertically, or diagonal. The first player who does this wins! But watch out when you're playing, as your opponent will try to connect their four chips as well!",
+    "developers": [
+        {
+            "githubUserName": "IgnaceMaes"
+        }
+    ],
+    "releaseData": "2022-10-22",
+    "links": {
+        "appstore": "https://apps.apple.com/app/four-in-a-row-classic/id1495540053",
+        "playstore": "https://play.google.com/store/apps/details?id=com.ignacemaes.fourinarow",
+        "website": "https://ignacemaes.com/"
+    }
```

Comment:
Is it possible to link to the project directly? Because this is your personal website

Reply:
The project doesn't have a stand alone website. Removed the `website` for now.

Reply:
Alright, it is always possible to add it in the future.